### ISSUE TITLE
parsedAlternates error on vg call

### ISSUE THREAD

**Issue by jcoliver** [NCBI-Hackathons/Graph_Genomes_CSHL#10]
```
REFFASTA="/home/data/reference/GRCh38_full_plus_hs38d1_analysis_set_minus_alts.fa"
VCFFILE="/home/devsci4/Graph_Genomes_CSHL/examples/vgInput.vcf.gz"
VGOUTFILE="/home/devsci4/Graph_Genomes_CSHL/examples/vgoutput.vg"
vg/bin/vg construct -r $REFFASTA -v $VCFFILE > $VGOUTFILE
```

Returns:

```
parsedAlternates: alignment does not start with match over padded sequence
11M2D9M
ZZZZZZZZZZQTTZZZZZZZZZZ
ZZZZZZZZZZQZZZZZZZZZ
```

**Comment by jcoliver** [comment]
Related to vcflib/vcflib#120 ?

**Comment by ajo2995** [comment]
looks like something weird is happening at chr1 71835. The alt column is empty. What were the inputs that produced vgInput.vcf.gz?

**Comment by jcoliver** [comment]
This was run on the uberbam.bam produced by an earlier version of scripts/fas2bam.pl. May have been resolved by c72035e. Once a machine is up & running we can try this again to see if the issue went away...

**Comment by nhansen** [comment]
I'll try to take a look at it today.

--Nancy
